src/main/webapp/app/layouts/navbar/navbar.component.html

```diff
@@ -71,13 +71,14 @@
     <ul class="navbar-nav ml-auto">
         <!-- jhipster-needle-add-element-to-menu - JHipster will add new menu it-->
         <li ngbDropdown class="nav-item dropdown pointer navbar-icon" display="dynamic" routerLinkActive="active" [routerLinkActiveOptions]="{ exact: true }">
-            <a class="nav-link" routerLink="admin/user-management" routerLinkActive="active" id="admin-menu" data-cy="adminMenu">
+            <a class="nav-link" routerLink="account/user-info" routerLinkActive="active" id="admin-menu" data-cy="userInfo">
```

Comment:
Should be added another option, the current one is for administration management of users

Reply:
I inquired @sfdsilva about this issue. We came to the conclusion that the button attached is for the user-info page.
Probably in the future we should add another button (only visible to the admin's) fot the user-management page.

![image](https://user-images.githubusercontent.com/72458547/102923215-48590780-4487-11eb-83d0-0340a250568c.png)